test/setup_environment.sh

```diff
@@ -448,7 +448,6 @@ if [[ -n ${build_bitcoind} ]]; then
     popd
 
     # Do the build
-    ./autogen.sh
-    CONFIG_SITE=$PWD/depends/x86_64-pc-linux-gnu/share/config.site ./configure --with-incompatible-bdb --with-miniupnpc=no --without-gui --disable-zmq --disable-tests --disable-bench --with-libs=no --with-utils=no
-    make src/bitcoind
+    cmake -B build -DBerkeleyDB_INCLUDE_DIR:PATH="$PWD/depends/x86_64-pc-linux-gnu/include" -DBoost_INCLUDE_DIR:PATH="$PWD/depends/x86_64-pc-linux-gnu/include" -DLibevent_DIR="$PWD/depends/x86_64-pc-linux-gnu/libevent"  -DWARN_INCOMPATIBLE_BDB=ON -DWITH_MINIUPNPC=OFF -DBUILD_GUI=OFF -DWITH_ZMQ=OFF -DBUILD_TESTS=OFF -DBUILD_BENCH=NO
```

Comment:
```suggestion
    cmake -B build --toolchain depends/x86_64-pc-linux-gnu/toolchain.cmake -DBUILD_TESTS=OFF -DBUILD_BENCH=OFF
```

Reply:
`-DBUILD_GUI=OFF` is redundant when depends are built with `NO_QT=1` and so on.

Reply:
Cool, thanks @hebasto.

Reply:
Done